src/RuleSet/DeclarationBlock.php

```diff
@@ -40,12 +40,23 @@ public static function parse(ParserState $parserState, ?CSSList $list = null): ?
         $comments = [];
         $result = new DeclarationBlock($parserState->currentLine());
         try {
+            $selectors = [];
             $selectorParts = [];
             $stringWrapperCharacter = null;
+            $functionNestingLevel = 0;
+            $consumedNextCharacter = false;
             do {
-                $selectorParts[] = $parserState->consume(1)
-                    . $parserState->consumeUntil(['{', '}', '\'', '"'], false, false, $comments);
+                if (!$consumedNextCharacter) {
+                    $selectorParts[] = $parserState->consume(1);
+                }
+                $selectorParts[] = $parserState->consumeUntil(
+                    ['{', '}', '\'', '"', '(', ')', ','],
```

Comment:
To make this easier to understand, I propose moving this array either to a class constant or a well-named local variable. WDYT?